cardGame/cardGame/src/components/Modal.jsx

```diff
@@ -0,0 +1,67 @@
+import Portal from "./Portal";
+import styled from 'styled-components';
+
+const Modal = ({modalOn, setModalOn}) => {
+
+    const ModalClose =()=> {
+        setModalOn(false);
+        location.reload();
+    }
+
+    return (
+        <>
+        <Portal>
+            <Background  modalOn={modalOn}>
+                <StModal>
+                    <h1>mamma mia!</h1>
+                    <StCloseBtn onClick={()=>ModalClose()}>게임으로 돌아가기</StCloseBtn>
+                </StModal>
+            </Background>
+        </Portal>
+        </>
+    )
+}
+
+export default Modal
+
+const Background = styled.div`
+    display: ${(props) => (props.modalOn ? "flex":"none")};
```

Comment:
조건부 스타일링 아주 좋습니당!
요거는 props로 안받고 
```suggestion
    display: ${(modalOn) => (modalOn ? "flex":"none")};
```
이런식으로 써줘도 된대!

Reply:
오!!!!!!!! 완존 꿀팁 props는 언제나 참.... 어색햐요